### PR TITLE
[Resource] Refactor Instance Action

### PR DESCRIPTION
**Pull request by michaelkad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
=== RUN   TestAccIBMPIInstanceAction
--- PASS: TestAccIBMPIInstanceAction (2064.83s)
PASS

=== RUN   TestAccIBMPIInstanceActionHardReboot
--- PASS: TestAccIBMPIInstanceActionHardReboot (377.99s)
PASS
...
```

### CODE REVIEW

File: website/docs/r/pi_instance_action.html.markdown [hunk 1]
```diff
@@ -41,32 +44,33 @@ resource "ibm_pi_instance_action" "example" {
 
 The `ibm_pi_instance_action` provides the following [timeouts](https://www.terraform.io/docs/language/resources/syntax.html) configuration options:
 
-- **create** - The action on the instance is considered failed if no response is received for 15 minutes.
-- **Update** The update action on the instance is considered failed if no response is received for 15 minutes.
-
+* **create** - The action on the instance is considered failed if no response is received for 15 minutes.
+* **Update** The update action on the instance is considered failed if no response is received for 15 minutes.
```
Comment (ismirlia) [inline website/docs/r/pi_instance_action.html.markdown:48]:
```suggestion
* **update** The update action on the instance is considered failed if no response is received for 15 minutes.
```

File: website/docs/r/pi_instance_action.html.markdown [hunk 2]
```diff
@@ -22,7 +24,8 @@ resource "ibm_pi_instance_action" "example" {
 
 ```
 
-**Note**
+### Note
```
Comment (ismirlia) [inline website/docs/r/pi_instance_action.html.markdown:27]:
```suggestion
### Notes
```

File: website/docs/r/pi_instance_action.html.markdown [hunk 3]
```diff
@@ -4,13 +4,15 @@ subcategory: "Power Systems"
 layout: "ibm"
```
Comment (ismirlia) [inline website/docs/r/pi_instance_action.html.markdown:4]: remove the newline on line 2?

**Review by ismirlia — APPROVED** [review]

**Comment by michaelkad** [comment]
Opened in main repo.

**Pull request closed without merging**